src/Sarif.Converters/SemmleQlConverter.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.CodeAnalysis.Sarif.Converters
     /// <summary>
     /// Converts a log file from the Semmle format to the SARIF format.
     /// </summary>
-    internal class SemmleConverter : ToolFileConverterBase
+    internal class SemmleQlConverter : ToolFileConverterBase
```

Comment:
Aargh. I could have _sworn_ that wasn't the case, but I just [looked it up](https://msdn.microsoft.com/en-us/library/ms229043(v=vs.100).aspx) and you're right. Fixed.

---
In reply to: [121228572](https://github.com/Microsoft/sarif-sdk/pull/582#discussion_r121228572) [](ancestors = 121228572)